### PR TITLE
Removes contribution guideline on bitwise operator spacing

### PR DESCRIPTION
**Pull request by ohnopigeons**
Formalizes oranges' solution where nobody cares

### CODE REVIEW

**Comment by RemieRichards** [comment]
I stand by it being in the "enforced/not enforced" section, like I suggested, rather than just missing from the contrib guides.

**Comment by nicbn** [comment]
Arithmetical operators are spaced, so bitwise should be too. Matter of consistency.

**Comment by RemieRichards** [comment]
That relies on the user mentally grouping bitwise and arithmetic operators together.
Which I think is wrong.
The only thing they share is that they're both binary (2 operand) operators.

**Comment by ChangelingRain** [comment]
Except for |, which I think is clearer if not spaced since it would resemble ||.

**Comment by nicbn** [comment]
Checked some C style coding guides, none of them tell to avoid or use spaces with bitwise. Well, maybe just make it not enforced then, if not even Google ([link](https://google.github.io/styleguide/cppguide.html)) cares about it, why should we?

**Comment by ohnopigeons** [comment]
The entire premise of this PR was to not care

**Comment by RemieRichards** [comment]
@ohnopigeons yes, and we handle that by putting things in the section explicitly designed for not caring, rather than just omitting them from the guidlines.

**Comment by ohnopigeons** [comment]
Enforced not enforced is slightly different from not caring in that you care that no one else cares.

I'll add it to the Enforced Not Enforced section a bit later if there's no opposition.

**Comment by RemieRichards** [comment]
there won't be opposition, I'm god.

**Comment by ohnopigeons** [comment]
Well I can't say no to god

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -218,7 +218,9 @@ The following different coding styles are not only not enforced, but it is gener
 * English/British spelling on var/proc names
 	* Color/Colour nobody cares,
 * Spaces after control statements
-	* if() if () nobody cares.
+	* if() if () nobody cares,
+* Spaces separating bitwise operators
+	* A|B A | B nobody cares.
```
Comment (lzimann) [inline .github/CONTRIBUTING.md]: We kinda care about `|` though.

Comment (ohnopigeons) [inline .github/CONTRIBUTING.md]: > we don't care
> we kinda care

Jesus christ

**Comment by nicbn** [comment]
A|B we care
A&B we don't care
A^B we don't care

**Comment by ohnopigeons** [comment]
> different care rules for bitwise operators
No

**Comment by Cyberboss** [comment]
You act like I closed the original PR for no raisin. Now you see the reason

**Comment by RemieRichards** [comment]
The reason I said to put it in the don't care section was to allow the individual to decide.
While lots of us may agree on ```A|B``` we couldn't agree on a general "spaces or no spaces" rule.
So just say that we don't particularly care about spacing in bitwise operators, but DON'T use any of those operators in the explainer.

eg:

"We couldn't come to a consensus on whether to space bitwise operators from their operands, so instead simply use your best judgement/preference"
Instead of:
"A|B or A | B! who cares? A&B? FUCK IT! A ^ B TOTALLY NOT CARING BRUDDA"

**Comment by ohnopigeons** [comment]
Then it doesn't belong in enforced not enforced

**Comment by RemieRichards** [comment]
Yes it does, holy shit.
That's WHAT THAT SECTION IS FOR.

**Comment by ohnopigeons** [comment]
If you can't say "A|B or A | B! who cares? A&B? FUCK IT! A ^ B TOTALLY NOT CARING BRUDDA" then it doesn't

Cyberboss was right

**Pull request closed without merging**